examples/COUPLE/fortran/libfwrapper.c

```diff
@@ -1,7 +1,7 @@
 /* ----------------------------------------------------------------------
    LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
-   www.cs.sandia.gov/~sjplimp/lammps.html
-   Steve Plimpton, sjplimp@sandia.gov, Sandia National Laboratories
+   https://www.lammps.org/, Sandia National Laboratories
+   The LAMMPS Developers, developers@lammps.org
```

Comment:
i'd prefer this line be like this everywhere:
LAMMPS development team: developers@lammps.org